cargo-insta/src/cli.rs

```diff
@@ -421,6 +421,7 @@ fn load_snapshot_containers<'a>(
             snapshot_containers.push((snapshot_container?, None));
         }
     }
+    snapshot_containers.sort_by(|(a, _), (b, _)| a.snapshot_path().cmp(&b.snapshot_path()));
```

Comment:
This is incredibly helpful for multiple snapshots that come from the same test.

However, it shows them in an odd order:
```
  tests/ui.rs (ui_test_err_missing_files-2)
  tests/ui.rs (ui_test_err_missing_files-3)
  tests/ui.rs (ui_test_err_missing_files-4)
  tests/ui.rs (ui_test_err_missing_files)
```

As a plus, it also groups together snapshots from functions with the same prefix.